packages/components-css/card-group/index.scss

```diff
@@ -0,0 +1,28 @@
+.frameless-card-group {
+  container-type: inline-size;
```

Comment:
Er wordt hier een `container-type` gezet, maar de code gebruikt verder `media` is het nog niet gelukt om in plaats van `media` queries `container` te gebruiken?

Reply:
Hij zat inderdaad op de card. Om dit mogelijk te maken, moet ik een extra laag in de card group index toevoegen rondom de frameless-card-group class. Hierdoor kan hij de elementen oppakken en stylen met container queries. Ik heb dit gedaan, maar vraag me af of dit de toegankelijkheid van de website vermindert. Waarom geen directe media queries gebruiken?